pom.xml

```diff
@@ -170,21 +175,30 @@
                 <version>${fasterxml-data-format.version}</version>
             </dependency>
 
-            <!-- jupiter -->
+            <dependency>
```

Comment:
guava is not jupiter -> wrong comment

Reply:
sure. it's mess with pom.xml after few merges